src/opensynth/models/faraday/model.py

```diff
@@ -324,7 +324,6 @@ def __init__(
         devices: int = 1,
```

Comment:
Note that we are not using `max_iter` anymore, this is superseded by `gmm_max_epochs`

Reply:
I've now removed references to `max_iter` 🙂